scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -11,10 +11,18 @@
 #  dbsnp, dbsnp_index: optional database of known germline variants
 #  cosmic, cosmic_index: optional database of known somatic variants
 #  is_run_orientation_bias_filter: if true, run the orientation bias filter post-processing step
-
+#  is_run_oncotator: if true, annotate the M2 VCFs using oncotator.  Important:  This requires a docker image and should
+#   not be run in environments where docker is unavailable (e.g. SGE cluster on a Broad on-prem VM).  Access to docker
+#   hub is also required, since the task will download a public docker image.
+#
+#
+# This WDL needs to decide whether to use the ``gatk_jar`` or ``gatk_jar_override`` for the jar location.  As of cromwell-0.24,
+#   this logic *must* go into each task.  Therefore, there is a lot of duplicated code.  This allows users to specify a jar file
```

Comment:
This will not work on docker + cloud backends.  As soon as you make the output a String, the file will not travel to downstream tasks if it is an override jar.

Not done.